etc/picongpu/crusher-ornl/batch_pipe.tpl

```diff
@@ -95,6 +96,29 @@ export OMP_NUM_THREADS=!TBG_coresPerGPU
 
 echo 'Start job with !TBG_nodes_adjusted nodes. Required are !TBG_nodes nodes.'
 
+if (( !TBG_nodes < 2 )); then
+    cat << EOF >&2
+Frontier does not reliably initialize its networking capabilities for
+single-node jobs. For a reliable streaming implementation, please adjust the
+.cfg file such that the job uses at least two nodes.
+EOF
+    exit 1
+fi
+
+cat << EOF >&2
```

Comment:
I think this second comment is something we usually put in the system's profile. Would you please do that? Together with a hint that ADIOS2 needs to be self-compiled in order to use SST?

Reply:
Should we really bother every user on Frontier with that note? Putting it in here means that only those that want to use streaming will see it (and also has the advantage that every workaround for streaming on Frontier stays in one file).

Reply:
I will merge it as it is, I think only a handful of users will use streaming.